misc-utils/findmnt.c

```diff
@@ -1880,6 +1993,10 @@ int main(int argc, char *argv[])
 			"options --target and --source can't be used together "
 			"with command line element that is not an option"));
 
+	if ((findmnt.flags & FL_POLL) && findmnt.filter)
+		errx(EXIT_FAILURE, _(
+			"options --poll and -Q,--filter can't be used together "));
+
```

Comment:
This is unnecessary. Just add `{ 'Q', 'p' }` to `ul_excl_t excl[]` .

Reply:
(Fixed locally).

Reply:
After updating as you show, findmnt accepts -Q and -p:
```
[yamato@dev64]~/var/util-linux/util-linux% ./findmnt -p -Q 'FSTYPE == "xfs"'
ACTION     TARGET SOURCE    FSTYPE OPTIONS
umount     /tmp/b tmpfs[/a] tmpfs  rw,nosuid,nodev,seclabel,nr_inodes=1048576,inode64^C
```

Am I missing something?

Reply:
See the `ul_excl_t excl[]`, there is a note ` /* rows and cols in ASCII order */`. You have to add `{ 'Q', 'p' }` after `{ 'P','l','r','x' }`.

Reply:
Oh, I see. I must be more careful. Fixed.